### ISSUE TITLE
[Change request] (Make VSCode warning easier to address)

### ISSUE THREAD

**Issue by polina-c** [dart-code-checker/dart-code-metrics#1221]
### DCM version

Can this question contain information how to get DCM version? As described below it is different in different sources.

### What problem do you want to solve?

If this warning contained instruction or link to instruction on how to update license, it would be easier to follow:

<img width="451" alt="Screenshot 2023-04-13 at 10 26 39 AM" src="https://user-images.githubusercontent.com/12115586/231839147-7939ddf7-a7de-4af6-a4a3-8ef313709a04.png">

DCM version is different in different places:

<img width="473" alt="Screenshot 2023-04-13 at 10 31 26 AM" src="https://user-images.githubusercontent.com/12115586/231838695-29e45153-3681-4fd3-8f1f-ccf6a8250b3e.png">

```
dcm --version
Dart Code Metrics version: 1.3.0
```

### What do you think is the correct solution?

-

### Participation

- [ ] I am willing to submit a pull request for this change.

### Additional comments

-
